ml-proto/test/resizing.wast

```diff
@@ -20,32 +14,26 @@
     (func $store_at_page_size (result i32) (i32.store (i32.const 0x10000) (i32.const 3)))
 
     (export "grow" $grow)
-    (func $grow (param $sz i32) (result i32)
-      (grow_memory (call $round_up_to_page (get_local $sz)))
-    )
-
-    (export "size_at_least" $size_at_least)
-    (func $size_at_least (param i32) (result i32) (i32.ge_u (memory_size) (get_local 0)))
+    (func $grow (param $sz i32) (result i32) (grow_memory (get_local $sz)))
```

Comment:
[AstSemantics](https://github.com/WebAssembly/design/blob/master/AstSemantics.md#resizing) says this should be a multiple of the page size, but this sends in values like 1 which are meant to be multiplied by it. Which is wrong?